Example/Podfile.lock

```diff
@@ -1,33 +1,40 @@
 PODS:
-  - Differentiator (3.0.1)
-  - RxCocoa (4.0.0):
+  - Differentiator (3.0.2)
```

Comment:
Same question here - I don't see a reason to update pods for a minor code change

Reply:
I agree. The file was updated when installing the pods localy. We can skip the changes.